hathor/consensus/poa/poa.py

```diff
@@ -43,9 +43,23 @@ def get_hashed_poa_data(block: PoaBlock) -> bytes:
     return hashed_poa_data
 
 
+def get_active_signers(settings: PoaSettings, height: int) -> list[bytes]:
+    """Return a list of signers that are currently active considering the given block height."""
+    active_signers = []
+    for signer_settings in settings.signers:
+        start_height = float('-inf') if signer_settings.start_height is None else signer_settings.start_height
```

Comment:
`start_height` will never be `None`.

Reply:
Thanks! Fixed in https://github.com/HathorNetwork/hathor-core/pull/1075/commits/26a1816603b23e293099b3cd86352174d5c9ed83

Reply:
Thanks!